bin/apostrophe

```diff
@@ -4,9 +4,32 @@ require('shelljs/global');
 require('colors');
 var program = require('commander');
 var util = require('../lib/util');
-var pkginfo = require('pkginfo')(module, 'version');
+var fs = require('fs');
 
-program.version(module.exports.version);
+program.version(getVersion());
+
+// Get version of apostrophe-cli and apostrophe.
+function getVersion() {
+  var pkginfo = require('pkginfo')(module, 'version');
+  var cwd = process.cwd();
+
+  var output = 'apostrophe-cli: ' + module.exports.version;
+  var aposPath = cwd + '/node_modules/apostrophe/';
+
+  // Append message for apostrophe.
+  if (fs.existsSync(aposPath)) {
+    var package = require(aposPath + 'package.json');
+
+    output += 'apostrophe v' + package.version + ' is installed in this project.';
+  } else {
+    var execSync = require('child_process').execSync;
+    var aposVersion = execSync('npm view apostrophe version').toString();
+
+    output += 'apostrophe v' + aposVersion.trim() + ' (latest) will be installed with a new project.';
```

Comment:
One last nitpick... just add:

"unless the boilerplate project's dependencies specify otherwise."

Then we're good to go thank you